dags/openshift_nightlies/tasks/utils/defaults.json

```diff
@@ -0,0 +1,26 @@
+{
+    "utils": [
+        {
```

Comment:
can we remove prometheus as a default and just have must-gather for now?

Reply:
done